src/components/Header.js

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+// nodejs library that concatenates classes
+import classNames from "classnames";
+// nodejs library to set properties for components
+import PropTypes from "prop-types";
```

Comment:
Hmm... this is also not really relevant for us since we don't use PropTypes in this project...